CHANGELOG.md

```diff
@@ -1,16 +1,23 @@
 # Mattermost Mobile Apps Changelog
 
-## 1.16.1 Release
-- Release Date: February 21, 2019
+## 1.17.0 Release
```

Comment:
We should highlight that we upgraded to RN 0.58.5 and that the iOS Share Extension is now built natively

Reply:
Oh and btw this should be on top of the 1.16.1 release